packages/apps/human-app/server/src/integrations/escrow/escrow-utils-gateway.service.ts

```
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common';
+import { ChainId, EscrowUtils } from '@human-protocol/sdk';
+
+@Injectable()
+export class EscrowUtilsGateway {
+  async getExchangeOracleAddressByEscrowAddress(
+    address: string,
+  ): Promise<string> {
+    const escrowsData = await EscrowUtils.getEscrow(
+      ChainId.POLYGON_AMOY,
+      address,
+    );
+    return escrowsData.exchangeOracle || '';
```

this should throw exception if exchangeOracle is not found